src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4445,6 +4445,13 @@ .ForwardHelpCategory Cmdlet
             new SessionStateVariableEntry(SpecialVariables.FirstToken, null, string.Empty),
             new SessionStateVariableEntry(SpecialVariables.StackTrace, null, string.Empty),
 
+            // Variable which controls the output rendering
+            new SessionStateVariableEntry(
+                SpecialVariables.PSStyle,
+                PSStyle.Instance,
+                RunspaceInit.PSStyleDescription,
+                ScopedItemOptions.AllScope),
```

Comment:
`AllScope` means this varaible will be copied to every new scope when that scope is created. Maybe that's not needed?

Reply:
will change to None